docs/integrate/lineage/marquez/index.md

```diff
@@ -0,0 +1,258 @@
+
+## Data lineage with CrateDB
+
+Today I want to share an example of how we can run Airflow DAGs against a CrateDB database and view lineage data.
+
+### Setup
+
+We will need Docker Compose v2, to install it, make it available to all users in the system, check the installed version, or upgrade from v1, please refer to [the Docker documentation on this topic].
+
+Let's now start Marquez:
+
+```bash
+git clone https://github.com/MarquezProject/marquez && cd marquez
+sudo ./docker/up.sh
+```
+
+While Marquez starts, let's open another terminal and proceed installing the Astro CLI for Airflow:
+
+```bash
+curl -sSL install.astronomer.io | sudo bash -s
+```
+
+Let's initialize a project folder:
+
+```bash
+mkdir datalineageeval
+cd datalineageeval
+astro dev init
+```
+
+We will use the PostgreSQL wire protocol to connect to CrateDB:
+
+```bash
+echo 'apache-airflow-providers-postgres' >> requirements.txt
+```
+
+Let's now configure Airflow to use Marquez as the lineage repository and connect to it via the Docker bridge interface:
+
+```bash
+cat <<EOF >>.env
+OPENLINEAGE_URL=http://172.17.0.1:5000
+OPENLINEAGE_NAMESPACE=example
+EOF
+```
+
+We will use the Airflow web interface, if you are running this on a headless system where you do not have access to a web browser you can run:
+
+```bash
+astro config set airflow.expose_port true
+``` 
+
+Astro also needs an internal PostgreSQL instance, but port 5432 is taken by Marquez's own internal database, so we will configure Astro to use port 5435 for its internal database:
+
+```bash
+astro config set postgres.port 5435
+sudo astro dev start
+```
+
+And we will start a single-node local CrateDB instance using port 5436 for the PostgreSQL wire protocol interface:
+
+```bash
+sudo docker run -d --name cratedb --publish=4200:4200 --publish=5436:5432 --env CRATE_HEAP_SIZE=1g crate/crate:5.9.5 -Cdiscovery.type=single-node
+```
+
+(NB this will return immediately once the image is downloaded but CrateDB may take a few seconds to start)
+
+
+Let's now use the CrateDB CLI to prepare the tables we will use for this example:
+
+```bash
+sudo apt install python3-pip
+sudo pip install crash
+crash
+```
+
+```sql
+CREATE TABLE public.Customers (
+	CustomerID TEXT PRIMARY KEY NOT NULL DEFAULT gen_random_text_uuid()
+	,CustomerName TEXT NOT NULL
+	,Country TEXT	
+	);
+	
+CREATE TABLE public.Invoices (
+	InvoiceID TEXT PRIMARY KEY NOT NULL DEFAULT gen_random_text_uuid()
+	,date TIMESTAMP DEFAULT now()
+	,CustomerID TEXT	
+	);
+	
+CREATE TABLE public.Products (
+  ProductID TEXT PRIMARY KEY NOT NULL DEFAULT gen_random_text_uuid(),
+  Description TEXT,
+  applicable_tax_percentage REAL
+);
+
+CREATE TABLE public.Invoice_items (
+  InvoiceID TEXT,
+  ProductID TEXT,
+  quantity  SMALLINT,
+  unit_price REAL,
+  PRIMARY KEY (InvoiceID,ProductID)
+);
+```
```

Comment:
Just a nit: The DDL statements bear deviances in formatting. Possibly adjust and align them?